packages/sirius-web/backend/sirius-web-application/src/main/resources/schema/siriusweb.graphqls

```diff
@@ -1,6 +1,6 @@
 extend type Viewer {
   project(projectId: ID!): Project
-  projects(page: Int!, limit: Int!): ViewerProjectsConnection!
```

Comment:
I would have expected `useProjects.ts` to be modified to take this GraphQL API change into account

Reply:
As expected the frontend on the project browser view displays an error: unknown argument `page` and `limit`

Reply:
Done